gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=org.web3j
-version=4.9.5-SNAPSHOT
```

Comment:
Do we need to change 4.10.0-SNAPSHOT ?

Reply:
When I will bump the version to 4.10.1...only for release we remove snapshot. This confusion is because till now it was a missmatch between the number of Snapshot version. If we released 4.9.5 we kept 4.9.5-SNAPSHOT, but this was wrong because next snapshot version should have been 4.9.6-SNAPSHOT